doc/lua_api.txt

```diff
@@ -79,6 +79,12 @@ The game directory can contain the following files:
       e.g. `disallowed_mapgen_settings = mgv5_spflags`
       These settings are hidden for this game in the world creation
       dialog and game start menu.
+    * `author`: The author of the game. It only appears when downloaded from
+                ContentDB.
+    * `release`: The ID corresponding to the version of the mod. It only
```

Comment:
This should never be set by modders

Reply:
"Should only ever be set by ContentDB, as it is an internal ID used to track versions`